app/src/sharedTest/java/org/oppia/app/utility/DragViewAction.kt

```diff
@@ -0,0 +1,173 @@
+package org.oppia.app.utility
+
+import android.os.SystemClock
+import android.view.View
+import android.view.ViewConfiguration
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.PerformException
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.action.CoordinatesProvider
+import androidx.test.espresso.action.MotionEvents
+import androidx.test.espresso.action.PrecisionDescriber
+import androidx.test.espresso.action.Swiper
+import androidx.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed
+import androidx.test.espresso.util.HumanReadables
+import org.hamcrest.Matcher
+import org.hamcrest.core.AllOf.allOf
+
+/**
+ * The number of move events to send for each drag.
+ */
+private const val DRAG_STEP_COUNT = 10
+
+/**
+ * Length of time a drag should last for, in milliseconds.
+ */
+private const val DRAG_DURATION = 1500
```

Comment:
Prefer including the time unit in the name, e.g. ``DRAG_DURATION_MS``

Reply:
done